### ISSUE TITLE
reveice script

### ISSUE THREAD

**Issue by Morr1s** [tgalal/yowsup#2176]
Hey guys,

I'm trying to get the sample eco application to work https://github.com/tgalal/yowsup/wiki/Sample-Application . But I always get this error:
  File "run.py", line 13, in <module>
    from yowsup.layers.axolotl                     import YowAxolotlLayer
ImportError: cannot import name 'YowAxolotlLayer'

There is a comment that says that the layer files are outdated?!. But where can I get a working receive script. Thank you very much for your help:)

# Edit
I used the script from here: #1648 
It first produced the error:
  File "run.py", line 4, in <module>
    from layer                                     import EchoLayer
  File "/home/greenhouse/test/layer.py", line 21, in <module>
    from yowsup.layers.protocol_media.picture              import YowMediaPictureLayer              #descarga imagen
  File "/usr/local/lib/python2.7/dist-packages/yowsup2-2.5.2-py2.7.egg/yowsup/layers/protocol_media/picture.py", line 2, in <module>
    from yowsup import ProtocolTreeNode
ImportError: cannot import name ProtocolTreeNode
I got it now to work. I just deleted the line that produeced the error. I don't know what it was for and what's now broken, but it can still answer text messages and that's all I wanted :)

**Comment by Morr1s** [comment]
Sorry for closing and reopening the issue, but can somebody tell me how to fix the error and not just "fix" it like I did. Thank you alot for your help!

**Comment by coolnickname** [comment]
The AxolotlLayer is now devided into 3 different layers. I found the names but I don't know in which order they go, so it doesn't work. For now just use the other example of the run file.On 28 Jul 2017 10:37 pm, Morr1s <notifications@github.com> wrote:Sorry for closing and reopening the issue, but can somebody tell me how to fix the error and not just "fix" it like I did. Thank you alot for your help!
